module/zstd/zfs_zstd.c

```diff
@@ -695,6 +699,7 @@ zstd_init(void)
 	pool_count = (boot_ncpus * 4);
 	zstd_meminit();
 
+#if !defined(_STANDALONE)
```

Comment:
I understand that resources are limited in the bootloader environment by why single out these kstats?  They're relatively light weight and there are many other in the code base which are still enabled.  Would it instead make more sense to provide a mechanism which could be used to disable them all?

Reply:
I disabled them because that was easier than writing the shims for the items they depended on, but I've forgotten the details. Let me find them.

Reply:
I've reworked the standalone support for more of the FreeBSD spl, so I've removed this from zfs_zstd.c